src/Development/IDE/Core/Shake.hs

```diff
@@ -392,9 +395,12 @@ shakeRun IdeState{shakeExtras=ShakeExtras{..}, ..} acts =
                   let res' = case res of
                           Left e -> "exception: " <> displayException e
                           Right _ -> "completed"
+                      profile = case res of
+                          Right (_, Just fp) -> ", profile saved at file://" <> fp
```

Comment:
This isn’t going to produce a valid URI in general, if you want the URI (I guess to be able to click on it in VSCode?), you can use `filePathToUri'` or alternatively, just print out the filepath.